assets/locales/en.json

```diff
@@ -141,5 +141,13 @@
   "runGarbageCollectorDone": {
     "title": "Garbage collector",
     "message": "The garbage collector ran successfully."
+  },
+  "moveRepositoryFailed": {
+    "title": "Move repository",
+    "message": "Could not move the repository from { currDir } to { newDir }."
+  },
+  "cantAddIpfsToPath": {
+    "title": "IPFS on PATH",
```

Comment:
“IPFS on path”? Not sure why we’d uppercase “path” here.

Reply:
We're referring to the environment variable PATH so we usually put that in uppercase 😃

Reply:
Ah right, never mind!